openpower/package/hostboot/hostboot.mk

```diff
@@ -16,7 +18,24 @@ HOSTBOOT_INSTALL_TARGET = NO
 HOSTBOOT_ENV_VARS=$(TARGET_MAKE_ENV) \
     CONFIG_FILE=$(BR2_EXTERNAL)/configs/hostboot/$(BR2_HOSTBOOT_CONFIG_FILE) \
     OPENPOWER_BUILD=1 CROSS_PREFIX=$(TARGET_CROSS) HOST_PREFIX="" HOST_BINUTILS_DIR=$(HOST_BINUTILS_DIR) \
-    HOSTBOOT_VERSION=`cat $(HOSTBOOT_VERSION_FILE)`
+    HOSTBOOT_VERSION=`cat $(HOSTBOOT_VERSION_FILE)` 
+
+define HOSTBOOT_APPLY_PATCHES
+       if [ "$(BR2_OPENPOWER_POWER9)" == "y" ]; then \
```

Comment:
Buildroot has a mechanism for applying patches already. Why are you not using that?

(It is my opinion that you should not be applying patches at all, and instead be committing your patches to branches and pointing the build that hostboot branch).